libs/domain/product/src/mocks/src/mock-product.providers.ts

```diff
@@ -45,4 +47,8 @@ export const mockProductProviders: Provider[] = [
     provide: ProductMediaConfig,
     useValue: productMediaConfig,
   },
+  {
+    provide: AlternativeProductsListService,
```

Comment:
I had not captured this before, what about renaming the service to `ProductRelationService`?

Reply:
I guess `ProductRelationsListService` will be a better name